site/topics/booleans/booleans.rst

```diff
@@ -1,6 +1,59 @@
-*******************************
-Logic, Booleans, More Functions
-*******************************
+***************
+Predicate Logic
+***************
+
+Logic First
+===========
+
+* Sometimes things are ``True`` or ``False``
+* I'm betting you actually are pretty familiar with some LOGIC
+    * AND
+    * OR
+    * NOT
+
+.. admonition:: Activity
+
+    Write out the truth tables for the logical operations **AND**, **OR** and **NOT**.
+
+    Don't know what a 'logical operator' or 'truth table' is? No problem, Ask:
+
+    * Wikipedia
+    * Google
+    * Your neighbour
+    * Just don't ask me!
+
+
+* To make parts of the program *conditionally* executed, we need a *formal* way to describe conditions.
+* We need: logic.
+* Let's try some comparison:
+    >>> 19 == 87
+    False
+
+    >>> 5 == 5
+    True
+
+* Note that ``==`` is *comparison* while ``=`` is *assignment*. They are not the same! Python will punish you if you forget this!
+    * You'll all mess this up eventually
+    * Just be thankful this isn't C/C++
```

Comment:
Could frame this more positively. Mistakes need to be made in order to learn from them. Mentioning c/cpp is unnecessary.